src/main/java/com/incognia/common/AdditionalLocation.java

```diff
@@ -3,10 +3,14 @@
 import lombok.Builder;
 import lombok.Value;
 
+import java.time.Instant;
+
 @Value
 @Builder
 public class AdditionalLocation {
   Double lat;
   Double lng;
+  @Deprecated
   Long timestamp;
+  Instant collectedAt;
```

Comment:
I decided to mark the old field as deprecated and add the new here as this class is used by the user. Given that we are bumping the major due to the removal of `GetSignup`, should we take the opportunity and remove the old field too?